<?xml version="1.0"?>

<launch>
    <!-- Run the architecture's component and test it based on random-based stimulus. -->
    
    <node pkg="armor"
          type="execute"
          name="armor_service" 
 
          args="it.emarolab.armor.ARMORMainService"

    > </node>


    <node pkg = "ontological_surveyor"  
          type = "robot_state.py"
          name = "robot_state"
          output= "screen"

    > </node>

    <node pkg = "ontological_surveyor"  
          type = "choose_move.py"
          name = "choose_move_server"
          output = "screen"

    > </node>

        <node pkg = "ontological_surveyor"  
          type = "execute_move.py"
          name = "execute_move_server"
          output = "screen"
   
    > </node>

    <node pkg = "ontological_surveyor"  
      type = "home_surveilance_state_machine.py"
      name = "surveyor_state_machine"
      output = "screen"

    > </node>

</launch>
